commit 511d4014beff9f79aeab1c0013470f21ac6b7df7
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Wed Jan 4 14:13:19 2017 +0100

    Fix dependencies, sort and clean trailing whitespaces
    
    Add missing KF5JobWidgets
    Drop unused KF5TextEditor

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e11d998..6fab26b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -195,33 +195,33 @@ set(REQUIRED_KF5_VERSION "5.7.0")
 
 find_package(KF5 ${REQUIRED_KF5_VERSION} REQUIRED
     COMPONENTS
-        Archive 
-        Codecs 
+        Archive
+        Codecs
         Completion
-        Config 
-        ConfigWidgets 
+        Config
+        ConfigWidgets
         CoreAddons
         DBusAddons
         DocTools
-        GuiAddons 
-        I18n 
-        IconThemes 
+        GuiAddons
+        I18n
+        IconThemes
         ItemViews
+        JobWidgets
+        KCMUtils
         KDELibs4Support
-        KIO 
-        Kross 
-        Parts 
+        KIO
+        Kross
+        Notifications
+        NotifyConfig
+        Parts
         Sonnet
-        TextEditor 
-        TextWidgets 
+        TextWidgets
         ThreadWeaver
-        Wallet 
-        WidgetsAddons 
+        Wallet
+        WidgetsAddons
         WindowSystem
-        XmlGui 
-        NotifyConfig
-        Notifications
-        KCMUtils
+        XmlGui
 )
 
 find_package(KF5Activities)
